packages/rollup-plugin-webbundle/README.md

```diff
@@ -70,22 +70,35 @@ import * as wbnSign from 'wbn-sign';
 import dotenv from 'dotenv';
 dotenv.config({ path: './.env' });
 
-const key = wbnSign.parsePemKey(process.env.ED25519KEY);
-
-export default {
-  input: 'src/index.js',
-  output: {
-    dir: 'dist',
-    format: 'esm',
-  },
-  plugins: [
-    webbundle({
-      baseURL: new wbnSign.WebBundleId(key).serializeWithIsolatedWebAppOrigin(),
-      static: { dir: 'public' },
-      output: 'signed.swbn',
-      integrityBlockSign: { key },
-    }),
-  ],
+export default async () => {
+  const key = wbnSign.parsePemKey(
+    process.env.ENC_ED25519KEY,
+    await wbnSign.readPassphrase()
+  );
+
+  return {
+    input: 'src/index.js',
+    output: { dir: 'dist', format: 'esm' },
+    plugins: [
+      webbundle({
+        baseURL: new wbnSign.WebBundleId(
+          key
+        ).serializeWithIsolatedWebAppOrigin(),
+        static: { dir: 'public' },
+        output: 'signed.swbn',
+        integrityBlockSign: {
+          strategy: new wbnSign.NodeCryptoSigningStrategy(key),
+        },
+        headerOverride: {
+          'cross-origin-embedder-policy': 'require-corp',
+          'cross-origin-opener-policy': 'same-origin',
+          'cross-origin-resource-policy': 'same-origin',
+          'content-security-policy':
+            "base-uri 'none'; default-src 'self'; object-src 'none'; frame-src 'self' https:; connect-src 'self' https:; script-src 'self' 'wasm-unsafe-eval'; img-src 'self' https: blob: data:; media-src 'self' https: blob: data:; font-src 'self' blob: data:; require-trusted-types-for 'script';",
```

Comment:
Could you update this to the latest CSP values from here https://github.com/WICG/isolated-web-apps? (mainly `frame-src` has been changed, I think).

Reply:
I assume, I should also update the code then?